README.md

```diff
@@ -239,8 +239,8 @@ The creators of Unikraft have built [KraftCloud](https://kraft.cloud): a next ge
 Unikraft is open-source and licensed under `BSD-3-Clause` and the copyright of its
 authors.  If you would like to contribute:
 
-1. Read the Developer Certificate of Origin Version 1.1 (https://developercertificate.org/)
-2. Signing all commits to the Unikraft project.
+1. Read the [Developer Certificate of Origin Version 1.1](https://developercertificate.org/).
```

Comment:
Perhaps it would be worth to also request signing off commits as described in the DCO.

Reply:
@michpappas , fixed.